src/components/Quotes/Quotes.js

```diff
@@ -1,33 +1,34 @@
 import React from 'react';
 import styles from './Quote.css';
-import { useQuotes } from '../../hooks/quotes';
+import { useQuotes, searchQuotes } from '../../hooks/quotes';
 
 const Quotes = () => {
-  const { handleClick, handleNumQuotesChange, searchQuotes, numQuotes, quotes } = useQuotes();
+  const { handleClick, handleNumQuotesChange, numQuotes, quotes } = useQuotes();
 
   return (
     <>
       <div className={styles.Quote}>
         <button onClick={() => handleClick('https://ron-swanson-quotes.herokuapp.com/v2/quotes/1')}>Get Random Quote</button>
```

Comment:
You're already using this url in your quotesApi.js file. you should refactor your handleClick to instead make a call to the api